ccd-definition/CaseEventToFields/WorkAllocationIntegrationFields-CUI-caseworkerEvents-nonprod.json

```diff
@@ -0,0 +1,107 @@
+[
```

Comment:
Should this file have CUI in the name ? I thought that was a different release ?

Reply:
@ss-evoco If you look at build-release-ccd-definition.sh script file, excludedFilenamePatterns looks for either prod/non-prod at the end of the file name. So CUI doesn't really matter. We create new files just to logically split them for different releases. I am still have to remove the word CUI if it bothers you :)

Reply:
@ss-evoco Updated as discussed